### DISCUSSION TITLE
Can I link my repo with my godaddy hosting server?

### DISCUSSION THREAD

**Discussion by rwaqas** [community/community#22632] [New to GitHub]
<p>Hello Guys,</p>
<p>Searching for a mechanism with which if a code is updated on github, than github automatically synchronize changes/code on to our godaddy server.</p>
<p>Is this or something like this, possible?</p>
<p>Any help will be highly appreciated.</p>
<p>Thanks</p>

**Comment by mpboom** [comment, marked as answer]
<p>It’s definitely possible. You can always just have a script somewhere check for changes and then uploading it trough FTP, for example.</p>
<p>However, that may not be a very effecient way to do it. The best thing you could probably do is contact GoDaddy or ask on a forum there if this is functionality they support. Otherwise you will be stuck with the solution above.</p>

**Comment by facundofarias** [comment]
I know you are trying to use a Github Action, but maybe you can give [DeployHQ](https://www.deployhq.com/) a try!
